README.md

```
@@ -44,28 +44,40 @@ HTTP API
+| **fen**      | string | required      | FEN of the position with at least the board part                                                |
+| **orientation** | string | white         | `white` or `black`                                                                              |
+| **size**     | int    | 360           | The width and height of the image                                                               |
+| **lastMove** | string | *(none)*      | The last move to highlight, e.g., `f4g6`                                                        |
+| **check**    | string | *(none)*      | A square to highlight for check, e.g., `h8`                                                     |
+| **arrows**   | string | *(none)*      | Draw arrows and circles, e.g., `Ge6g8,Bh7`, possible color prefixes: `G`, `B`, `R`, `Y`         |
+| **squares**  | string | *(none)*      | Marked squares, e.g., `a3,c3`                                                                   |
+| **coordinates** | bool   | *false*       | Show a coordinate margin                                                                        |
+| **colors**   | string | lichess-brown | Theme: `wikipedia`, `lichess-brown`, `lichess-blue`, `random` (generate one on the fly)         |
+| **pieceSet** | string | required      | See, e.g., `merida`, `alpha`, `leipzig`, `alpha`, `california`, `celtic`, `companion`, `fantasy`, `governor`, `kiwen-suwi`, `letter`, `merida`, `mpchess`, `reillycraig`, `spatial`, `anarcandy`, `cardinal`, `chess7`, `cooke`, `fresca`, `horsey`, `kosal`, `libra`, `monarchy`, `pirouetti`, `riohacha`, `staunty`, `caliente`, `cburnett`, `chessnut`, `dubrovny`, `gioco`, `icpieces`, `leipzig`, `maestro`, `mono`, `pixel`, `shapes`, `tatiana`, `random` (generate one on the fly) |
+
+### `GET /board.png` render a PNG
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)